examples/abitest/module_1/rust/src/lib.rs

```diff
@@ -26,12 +26,14 @@ use protobuf::ProtobufEnum;
 // is what's sent back to the frontend.
 #[no_mangle]
 pub extern "C" fn backend_oak_main(handle: u64) -> i32 {
-    match std::panic::catch_unwind(|| main(handle)) {
-        Ok(rc) => rc,
-        Err(_) => oak::OakStatus::ERR_INTERNAL.value(),
-    }
+    std::panic::catch_unwind(|| main(handle))
```

Comment:
Nit: Having to do a double unwrap unfortunately feels like somewhat of a regression

Reply:
Yeah it's not great, but `catch_unwind` returns a `Result`, which wraps the `Result` coming from the `main` function. I think it's fine, but if we feel it gets too hard to reason about we can consider alternatives.